src/common/libflux/handle.c

```diff
@@ -238,6 +240,43 @@ static connector_init_f *find_connector (const char *scheme, void **dsop)
     return NULL;
 }
 
+/* If flux.toml has "flux_uri" set, return its value (caller must free).
+ * Return NULL if not configured, or if errors are encountered while
+ * reading/parsing the config file, so flux_open() can try the next option.
+ */
+static char *lookup_cf_uri (void)
+{
+    const char *cf_path;
+    char buf[PATH_MAX + 1];
+    cf_t *cf;
+    const cf_t *val;
+    const char *s;
+    char *cpy;
+
+    if (!(cf_path = getenv ("FLUX_CF_PATH"))) {
+        int conf_flags = executable_is_intree () ? CONF_FLAG_INTREE : 0;
+        cf_path = flux_conf_get ("cf_path", conf_flags);
```

Comment:
These lines don't appear in coverage on codecov.io. If that is right, it might be nice to test behavior of `lookup_cf_uri()` wihtout `FLUX_CF_PATH` set.